### ISSUE TITLE
More vector instruction decoding issues

### ISSUE THREAD

**Issue by mxz297** [dyninst/dyninst#402]
@jdetter I carefully checked the binary that contains many vector instructions and caused many decoding errors. There are still quite some decoding mistakes, but we are much better than before. 

Below are a set of instructions that InstructionAPI reported as "No_Entry", but objdump shows otherwise.

```
5718aa:	62 f1 ed 48 fb d6    	vpsubq %zmm6,%zmm2,%zmm2
5c6972:	62 b1 3d 48 72 34 8e 03	 vpslld $0x3,(%rsi,%r9,4),%zmm8
61e150:	62 e2 7e 48 3a cb    	vpbroadcastmw2d %k3,%zmm17
62d197:	62 f2 6d 4a 64 c1    	vpblendmd %zmm1,%zmm2,%zmm0{%k2}
678edb:	c4 e3 79 0b c8 02    	vroundsd $0x2,%xmm0,%xmm0,%xmm1
6d3246:	62 b1 fd 08 2e c5    	vucomisd %xmm21,%xmm0
921129:	66 0f 3a 0a c0 02    	roundss $0x2,%xmm0,%xmm0
a1cfbb:	62 d2 f5 78 b8 d1    	vfmadd231pd {rz-sae},%zmm9,%zmm1,%zmm2
```

I think I have merged all your commits, but let me know if you are not able to reproduce any of these decoding issues.

**Comment by jdetter** [comment]
Still making progress on this one, should be fixed this weekend for sure.

**Comment by wrwilliams** [comment]
Thanks, guys.

**Comment by jdetter** [comment]
I made a lot of progress this weekend, the only instruction that's giving me quite a bit of trouble is `vpslld` because it's an overridden SSE group instruction, I'll keep working at it.

**Comment by jdetter** [comment]
@wrwilliams I had to add new instructions to the entry ids header file, so do these fixes need to be pushed to a major release or can they be put into a minor release? The fixes will change the ABI for dyninst.

**Comment by wrwilliams** [comment]
Minor but not patch.

**Comment by jdetter** [comment]
Hey guys, sorry I haven't had much time for this. I'll try to get this finished up this weekend. I converted the decoding debug to dyninst's debug system where you set an environment variable to enable debug messages so instead of commenting out a macro to enable debug you just have to set an environment variable. 

@mxz297 Any new decoding issues besides the ones in this issue?

**Comment by mxz297** [comment]
@jdetter No worries. I have not heard of any new decoding issues.

**Comment by mxz297** [comment]
This issue is still applicable to Dyninst 10.1. This will be a future release item.

**Comment by mxz297** [comment]
Capstone is able to correctly decode these instructions.

**Comment by mxz297** [comment]
Closed via #638
